js/libs/builder.js

```diff
@@ -324,6 +324,7 @@ new Vue({
       });
 
       Fliplet.Widget.toggleCancelButton(false);
+      Fliplet.Widget.setSaveButtonLabel('Save');
```

Comment:
I see a repetition here. Please use two functions instead of repeating the "Save" and "Save & Close" variable every time.

e.g.

```js
function setSaveLabel() {
  Fliplet.Widget.setSaveButtonLabel('Save');
}
```

or

```js
Fliplet.Widget.setSaveButtonLabel(saveLabel);
```

Let's please keep in mind our guidelines recommend keeping code **DRY**.

@tonytlwu anything else you want to add?

Reply:
@squallstar I'm not too fussed because it's a function with one line and one parameter. The abstraction doesn't add much for me. Happy to go with whatever you enforce here.

Reply:
I'm just worried that at some point we end up updating "Save" to "Something else" but don't update all instances.

If you're fine with Upplab's current implementation I'm ok with getting it past this little concern I have.

Reply:
@squallstar @tonytlwu I am sorry, I did not foresee that new ones can be added except of this.
I'll create constant dictionary now with "Save" and "Save & Close" and will use them as `Fliplet.Widget.setSaveButtonLabel(SAVE_BTN_LABELS.SAVE);`
and
`Fliplet.Widget.setSaveButtonLabel(SAVE_BTN_LABELS.SAVE_AND_CLOSE);`